vm/src/stdlib/os.rs

```diff
@@ -1102,6 +1118,34 @@ mod _os {
                 st_ctime_ns: to_ns(ctime).into(),
             }
         }
+
+        #[pyslot]
+        fn tp_new(_cls: PyTypeRef, args: FuncArgs, vm: &VirtualMachine) -> PyResult {
+            let flatten_args = |r: &[PyObjectRef]| {
+                let mut vec_args = Vec::from(r);
+                loop {
+                    if vec_args.len() == 1 {
+                        // SAFETY: len is check in previous line
+                        let obj = vec_args.first().unwrap();
```

Comment:
```suggestion
                    if let Some(obj) = vec_args.iter().exactly_one() {
```

Reply:
@youknowone Thanks for the suggestion! Will change this as suggested.